src/point_one/fusion_engine/messages/control.h

```diff
@@ -123,6 +123,8 @@ struct P1_ALIGNAS(4) ResetRequest : public MessagePayload {
   static constexpr uint32_t RESET_POSITION_DATA = 0x00000100;
   /** Delete all saved satellite ephemeris. */
   static constexpr uint32_t RESET_EPHEMERIS = 0x00000200;
+  /** Delete all GNSS time information. */
+  static constexpr uint32_t RESET_TIME = 0x00000004;
```

Comment:
This is in the wrong place. It should be in the "Runtime State Reset" group above with the other masks in its range.

Should we call this `RESET_GNSS_TIME` for clarity?